### ISSUE TITLE
FDS: Donkey Kong Luigi Edition (Hack)/Akumajou Dracula T+Eng

### ISSUE THREAD

**Issue by monsterro** [frederic-mahe/Hardware-Target-Game-Database#556]
Donkey Kong Luigi Edition (Hack) szemigi 2017-07-21.fds has a wrong checksum - .ips from https://www.romhacking.net/hacks/3584/ makes the patched file 16 bytes larger by adding 00s at the end. Checksum from the FDS pack is calculated from a file with first 16 bytes cut off, making the rom unusable.

Akumajou Dracula T+Eng v1 DvD Translations.fds - using ips from https://www.romhacking.net/translations/5543/ and following the instructions to the dot and removing header from the file gives different checksum than the one in the pack.

Diff attached
[FDS_diff.txt](https://github.com/frederic-mahe/Hardware-Target-Game-Database/files/6111981/FDS_diff.txt)

**Comment by tjanas** [comment]
internal FDS headers of the translation FDS in the pack as well as the selected nointro FDS with checksum 725c772e:
side A:
`01 2A 4E 49 4E 54 45 4E 44 4F 2D 48 56 43 2A A4 41 4B 4D 20 02 00 00 00 00 0F FF FF FF FF FF 61 12 17 49 61 00 00 02 06 5B 04 66 01 76 03 19 FF 80 31 09 09 02 00 00 00`
side B:
`01 2A 4E 49 4E 54 45 4E 44 4F 2D 48 56 43 2A A4 41 4B 4D 20 02 01 00 00 00 0F FF FF FF FF FF 61 12 17 49 61 00 00 02 00 11 02 29 00 76 03 19 FF 80 31 09 09 02 01 00 00`


I bet if you change your resulting FDS file, it will match what is in the pack

**Comment by Infinitum8t** [comment]
@monsterro 
Try these steps for Akumajou Dracula T+Eng v1 DvD Translations.fds:

1. Use the source file 7448fb40.
2. Use ROM Expander Pro (in the patch download), or following the manual hex edit instructions in the [README file](https://www.romhacking.net/translations/fds/patches/5543readme.txt) will produce a file 131,016 bytes with CRC32 14fd9412. 
3. Using this [ONLINE PATCHER](https://www.romhacking.net/patch/index.php?file=translations/5543&patch_name=Akumajou+Dracula+English+translation+vA&crc=F13BAEC1) or LunarIPS, will produce a file 131,016 bytes and c2c3a85c.
4. After removing the header, you will have your file of 131,000 bytes and 725c772e.

**Comment by Infinitum8t** [comment]
@tjanas and @monsterro 
Regarding "Donkey Kong Luigi Edition (Hack) szemigi 2017-07-21.fds", file 6689ac17 listed in SMDB, was arrived at by using the correctly patched file a4cddf64 and deleting the first 16 bytes of that file. The file size is 65,500 bytes. When tested, this file displayed the message "UNKNOWN ROM FORMAT".

The correct file should be a4cddf64 and file size is 65,516 bytes. This file was found working properly (at least for fifteen minutes). Both tests were conducted with an N8 Pro, on a Mini NT Noir. 
The file change occurred during #470.

**Comment by tjanas** [comment]
@monsterro @Infinitum8t I believe you are both correct. The original file of 65516 bytes with checksum `a4cddf64` (before it was truncated) was improperly sized to begin with, as it was already headerless, so it had 16 trailing garbage bytes (all headerless FDS disk images should have a size that is a multiple of 65500 bytes).

The correct file (truncating the trailing 16 bytes from checksum'd file `a4cddf64`) should have the following attributes:
```
Size: 65500 bytes
CRC32: 9ffccba3
SHA256: 08130011e7e41c43f4c65d8c4ef05309cec69f6722f72777caad9c3bbb44dd6c
SHA1: c3599b2d79ec738b5e8c11b0e4bf1d1b62914f5d
MD5: 4249f3754f3070d1a0f9bc17a983d5d8
```
